### PR TITLE
Update copy when full

### PR DESCRIPTION
**Pull request by Victa**

### CODE REVIEW

File: src/components/CheckoutBanner/CheckoutBanner.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ class CheckoutBanner extends React.Component {
     return (
       <div className="checkout-banner">
         <div className="subhead-1" id={`eventbrite-widget-modal-trigger-${id}-${uniqueID}`}>
-          October 4-8 FTW Retreat is almost sold out! <u>Click to reserve your spot</u>.
+          October 4-8 FTW Retreat is  sold out! <u>Add me to the waitlist</u>.
```
Comment (Victa) [inline src/components/CheckoutBanner/CheckoutBanner.jsx]: Could this trigger the mailchimp modal instead?

File: src/pages/index.js [hunk 2]
```diff
@@ -122,6 +122,8 @@ const IndexPage = () => {
                 </p>
               </div>
 
+              // Trigger mailchimp instead
+              // + Change copy to :add me to the waitlist:
               <CheckoutButton id={eventID} showSecondaryCTA={true} />
```
Comment (Victa) [inline src/pages/index.js]: Could this trigger the mailchimp modal instead?

File: src/pages/index.js [hunk 3]
```diff
@@ -369,6 +371,8 @@ const IndexPage = () => {
                 A picturesque drive just 1.5 hours from Montreal, the eight-bedroom, contemporary residence will be our homebase for the retreat. Watch the fall leaves spectacle from the deck, indulge in a soak in the outdoor hot tub, and get comfortable in the two seperate living areas. Next to the Mont Tremblant National Park, there is easy access next to gondola rides, hikes, bike trails and many water activities on Lake Tremblant.
               </p>
 
+              // Trigger mailchimp instead
+              // + Change copy to :add me to the waitlist:
               <CheckoutButton id={eventID} showSecondaryCTA={true} />
```
Comment (Victa) [inline src/pages/index.js]: Could this trigger the mailchimp modal instead?

**Comment by Victa** [comment]
We basically needs to change the copy of 3 buttons + trigger mailchimp instead of evernote.

**Comment by Victa** [comment]
- [ ] @kuanluo to update the FAQ + copy

then we are good to merge

**Comment by kuanluo** [comment]
A few copy updates: 

- [ ] change the homepage copy for the main body. The last paragraph edits:

You will be amazed by what can be revealed when you dedicate 5 days to connect with other badass women leaders in tech and embrace of your own truth. ~~Join us this October~~.

The October retreat is sold out. Join our newsletter to be receive future retreat announcements. 

- [ ] Is it possible to have "join our newsletter" in the above copy linked, so once clicked, the page auto-scrolls to the bottom of the page to the email input area? 

- [ ] FAQ update. Delete this question:

> How much does the retreat cost?
> The retreat costs between $1,250 and $1,750 depending on your room choices (most of the accommodations are private rooms with queen-size beds). It includes 4-night accommodations, nourishing meals every day and transportation to and from Montréal airport.

**Add a new one:** 
How do I register for the Fall retreat? 
Our October retreat is sold out. Join our newsletter to be receive future retreat announcements. Hope to see you on a future FTW Retreat!

**Comment by kuanluo** [comment]
@Victa @lesliepassante Thank you for working on the auto-scroll thing. I think scrolling to the bottom might make the most sense, since it's the same behavior on the homepage as well as FAQ.

**Pull request merged**